app/src/main/java/kr/ac/kaist/team888/hangulcharacter/HangulCharacter.java

```diff
@@ -46,6 +47,19 @@ protected HangulCharacter() {
 
     skeletons = gson.fromJson(data.getAsJsonArray(SKELETONS_KEY), collectionType);
     region = calculateRegion();
+
+    FeatureController.getInstance().registerOnFeatureChangeListener(
+        new FeatureController.OnFeatureChangeListener() {
+          @Override
+          public void onFeatureChange() {
+          }
+
+          @Override
+          public int getPriority() {
+            return 0;
```

Comment:
I think this `0` is the priority of the new listener. I suggest make `private` class member for priority and return it as the `getter` of it. What do you think?